### PR TITLE
Feature/core 921 fix special character handling

### PR DESCRIPTION
**Pull request by wulfraem**
- draft PR to run tests
- [CORE-921]

[CORE-921]: https://evannetwork.atlassian.net/browse/CORE-921

### CODE REVIEW

File: src/dfs/ipfs.spec.ts [hunk 1]
```diff
@@ -114,8 +105,48 @@ describe('IPFS handler', function test() {
...
+    it('should be able to add a file with umlauts, that have been encoded as utf8', async () => {
+      const content = 'öäüßÖÄÜ';
+      const encoding = 'utf8';
+      const hash = await ipfs.add('test', Buffer.from(content, encoding));
+      expect(hash).not.to.be.undefined;
+      const fileContent = await ipfs.get(hash);
+      expect(fileContent.toString(encoding)).to.eq(content);
+    });
+
+    it('should be not be able to add a file with extended unicodes (properly), that have been encoded as binary', async () => {
```
Comment (Philser) [inline src/dfs/ipfs.spec.ts]:
```suggestion
    it('should not be able to add a file with extended unicodes (properly), that have been encoded as binary', async () => {
```

File: src/dfs/ipfs.ts [hunk 2]
```diff
@@ -298,4 +298,17 @@ export class Ipfs extends Logger implements DfsInterface {
       }, 60 * 1000);
     }
   }
+
+  /**
+   * Tries to decode given Buffer to UTF-8, if this leads to invalid characters, decode to Latin-1.
```
Comment (karladler) [inline src/dfs/ipfs.ts]: doc says to latin-1 but implementation says binary?

Comment (karladler) [reply src/dfs/ipfs.ts]: ahhh okay :D 

> 'binary': Alias for 'latin1'.

File: VERSIONS.md [hunk 3]
```diff
@@ -4,6 +4,7 @@
 ### Features
 
 ### Fixes
+- fix buffer to-string conversion, try to decode to `utf8`, if this fails, decode it to `binary`
```
Comment (S3bb1) [inline VERSIONS.md]:
```suggestion
- fix buffer-to-string conversion, try to decode to `utf8`, if this fails, decode it to `binary`
```

**Pull request merged**